src/main/java/org/fundacionjala/pivotal/util/CommonActions.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.pivotal.util;
+
+import org.fundacionjala.pivotal.model.pageobjects.AbstractPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.support.ui.WebDriverWait'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)